webviz_subsurface/containers/_segy_viewer.py

```diff
@@ -0,0 +1,522 @@
+from uuid import uuid4
+from pathlib import Path
+import json
+
+import xtgeo
+
+
+import dash
+from dash.exceptions import PreventUpdate
+from dash.dependencies import Input, Output, State
+import dash_html_components as html
+import dash_core_components as dcc
+
+# pylint: disable=no-name-in-module
+from dash_colorscales import DashColorscales
+
+import webviz_core_components as wcc
+from webviz_config import WebvizContainerABC
+from webviz_config.webviz_store import webvizstore
+from webviz_config.common_cache import CACHE
+
+from ..datainput._seismic import load_cube_data, get_iline, get_xline, get_zslice
```

Comment:
Reorder import order according to https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Isnt it ordered?
standard => 3rdparty => local